terraform/modules/statuscake/main.tf

```diff
@@ -0,0 +1,13 @@
+resource statuscake_test alert {
+
+  website_name   = var.alerts["alert"]["website_name"]
```

Comment:
suggestion: you could use a similar pattern (`for_each`) used in other places
https://github.com/DFE-Digital/teacher-training-api/blob/master/terraform/modules/statuscake/resources.tf#L2